v3 error: flag_timestamp.go:92:44: undefined: time.TimeOnly

```
/go/pkg/mod/github.com/urfave/cli/v3@v3.0.0-beta1/flag_timestamp.go:92:44: undefined: time.TimeOnly
```
https://ci.codeberg.org/repos/13526/pipeline/30/6

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/urfave/cli/v3
 
-go 1.18
+go 1.22
 
 require github.com/stretchr/testify v1.9.0
 

